### ISSUE TITLE
rustc-nightly breaks with latest update of libllvm

### ISSUE THREAD

**Issue by ayushashi11** [termux/termux-packages#5519]
**Problem description**

rustc-nightly uses llvm version 9. But with the latest upgrade of libllvm from version 9 to 10, it breaks.
```
$pkg show rustc-nightly
Package: rustc-nightly
Version: 1.43.0-4
Maintainer: @its-pointless
Installed-Size: 539 MB
Depends: rustc-nightly
Homepage: https://www.rust-lang.org
Download-Size: 48.5 MB
APT-Manual-Installed: yes
APT-Sources: https://dl.bintray.com/xeffyr/unstable-packages unstable/main aarch64 Packages
Description: Rust compiler and utilities (nightly version)
```
**Steps to reproduce**

```
$rustc --version
CANNOT LINK EXECUTABLE "rustc": library "libLLVM-9.so" not found
```
**Expected behavior**

```
$rustc --version
1.43.0-nightly
```
**Additional information**

```
Packages CPU architecture:
aarch64
Subscribed repositories:
# sources.list
deb https://termux.org/packages/ stable main
# science-repo (sources.list.d/science.list)
deb https://dl.bintray.com/grimler/science-packages-24 science stable
# game-repo (sources.list.d/game.list)
deb https://dl.bintray.com/grimler/game-packages-24 games stable
# sources.list.d/pointless.list
deb https://its-pointless.github.io/files/24 termux extras
# x11-repo (sources.list.d/x11.list)
deb https://dl.bintray.com/xeffyr/x11-packages x11 main
# unstable-repo (sources.list.d/unstable.list)
deb https://dl.bintray.com/xeffyr/unstable-packages unstable main
Updatable packages:
libc++/stable 21d-1 aarch64 [upgradable from: 20]
termux-api/stable 0.50-2 aarch64 [upgradable from: 0.50-1]
Android version:
10
Kernel build information:
Linux localhost 4.4.177-18057978 #1 SMP PREEMPT Sat Jun 20 23:25:41 KST 2020 aarch64 Android
Device manufacturer:
samsung
Device model:
SM-A750F
```

**Comment by xeffyr** [comment]
Rustc nightly has build issues with LLVM 10.

**Comment by ayushashi11** [comment]
Can I see?

**Comment by Grimler91** [comment]
@ayatakesi https://github.com/termux/unstable-packages/runs/862893443

**Comment by its-pointless** [comment]
I will have a look and fix this soonish

**Comment by landfillbaby** [comment]
should have been fixed by termux/unstable-packages#185
